File: order_backend_springboot/src/main/java/com/order/domain/entity/Seat.java
package com.order.domain.entity;

import lombok.Data;
import org.hibernate.annotations.DynamicUpdate;

import javax.persistence.*;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

@Entity
@DynamicUpdate
@Data
public class Seat {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private Integer number;

    private String name;

    private String latlng;

    private String description;

    private Integer status;

    private Date createTime;

    private Date updateTime;

    public Seat() {
    }
}

File: order_backend_node/app/model/seat.js
'use strict';

module.exports = app => {
  const { INTEGER, STRING } = app.Sequelize;
  return app.model.define('seat', {
    id: { type: INTEGER, primaryKey: true, autoIncrement: true },
    number: INTEGER,
    name: INTEGER,
    latlng: STRING,
    description: STRING,
    status: INTEGER,
    del: INTEGER,
  });
};
